drivers/SmartThings/matter-switch/src/utils/switch_utils.lua

```diff
@@ -63,6 +63,14 @@ function utils.mired_to_kelvin(value, minOrMax)
   end
 end
 
+function utils.check_vendor_overrides(device, override_key)
```

Comment:
This doesn't necessarily return a bool, so maybe prefixing this with `check_` could be misleading in the future. Also, we are keying the product inside of a vendor, so perhaps`get_product_override_field` or similar?